### ISSUE TITLE
Documentation doesn't match code snippet and image  

### ISSUE THREAD

**Issue by gretchunkim** [SharePoint/sp-dev-docs#7838]
### What type of issue is this?

Documentation issue / typo

### What SharePoint development model, framework, SDK or API is this about?

💥 SharePoint Framework

### Target SharePoint environment

SharePoint Online

### What browser(s) / client(s) have you tested

- [X] 💥 Internet Explorer
- [X] 💥 Microsoft Edge
- [X] 💥 Google Chrome
- [X] 💥 FireFox
- [X] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

- browser version: n/a
- SPFx version: 1.14.0
- Node.js version: 14.17.1
- etc 

### Issue description

Two seperate issues:
**1. Wrong file extension type referenced in code snippet. Showing tsx code snippet instead of ts.**
![MS](https://user-images.githubusercontent.com/8564035/160735017-735f05e9-4728-44ff-92f2-fa64f41acc59.PNG)

**2. Sample image doesn't match html in the code snippet content.**  For instance, there is no class styles.button, title and all other elements don't match up with the image. 

![ms2](https://user-images.githubusercontent.com/8564035/160734888-9f538324-b215-4373-91f1-6c978cc86f15.PNG)
(https://user-images.githubusercontent.com/8564035/160734255-15848d38-3b75-40a4-8df0-510f647edde5.PNG)

One general question:   Can we get sample source code that are demonstrated from 'Getting started' as it will be helpful?

**Comment by deathstarjim0525** [comment]
I was just working through the getting started series as well and seeing the same discrepancies. It looks to me like there's some kind of style sheet missing maybe or something like that? Is that part of this repo so we can download and continue?

**Comment by Teka67** [comment]
If you have installed @microsoft/generator-sharepoint 1.14.0 version, then you have this problem. With version 1.13.1 you do not have this problem.

**Comment by Teka67** [comment]
The Material is not up to date.

**Comment by deathstarjim0525** [comment]
@Teka67  is there a process to update the content so that it matches? Would I do that through a pull request?

**Comment by bruzie** [comment]
Until the docs are updated (or if they need to be completely redone with 1.15), you can paste this CSS inside the .[webpartname] declaration in the SCSS file (generated from a 1.13.1 project):

```
.container {
    max-width: 700px;
    margin: 0px auto;
    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 25px 50px 0 rgba(0, 0, 0, 0.1);
  }

  .row {
    @include ms-Grid-row;
    @include ms-fontColor-white;
    background-color: $ms-color-themeDark;
    padding: 20px;
  }

  .column {
    @include ms-Grid-col;
    @include ms-lg10;
    @include ms-xl8;
    @include ms-xlPush2;
    @include ms-lgPush1;
  }

  .title {
    @include ms-font-xl;
    @include ms-fontColor-white;
  }

  .subTitle {
    @include ms-font-l;
    @include ms-fontColor-white;
  }

  .description {
    @include ms-font-l;
    @include ms-fontColor-white;
  }

  .button {
    // Our button
    text-decoration: none;
    height: 32px;

    // Primary Button
    min-width: 80px;
    background-color: $ms-color-themePrimary;
    border-color: $ms-color-themePrimary;
    color: $ms-color-white;

    // Basic Button
    outline: transparent;
    position: relative;
    font-family: "Segoe UI WestEuropean","Segoe UI",-apple-system,BlinkMacSystemFont,Roboto,"Helvetica Neue",sans-serif;
    -webkit-font-smoothing: antialiased;
    font-size: $ms-font-size-m;
    font-weight: $ms-font-weight-regular;
    border-width: 0;
    text-align: center;
    cursor: pointer;
    display: inline-block;
    padding: 0 16px;

    .label {
      font-weight: $ms-font-weight-semibold;
      font-size: $ms-font-size-m;
      height: 32px;
      line-height: 32px;
      margin: 0 4px;
      vertical-align: top;
      display: inline-block;
    }
  }
```

**Comment by andrewconnell** [comment]
@gretchunkim which doc was this in reference to? there's a LOT of tutorials, so would help if you can include a link to the actual doc in question

**Comment by gretchunkim** [comment]
@andrewconnell here's the exact link: 
https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/build-a-hello-world-web-part

That's the problem with issue template as well as in the MS documentation page in which it says to submit a view feedback for "This page", hence we assume that link is already included (but it's not).

**Comment by andrewconnell** [comment]
@gretchunkim said:

> That's the problem with issue template as well as in the MS documentation page in which it says to submit a view feedback for "This page", hence we assume that link is already included (but it's not).

Actually, it is... when you click that link it adds a bunch of stuff to the issue template that states "don't delete it". Here's what you get when you click that link for the page you linked to... if you left that stuff there, the link and the actual file would be included.

![Screen Shot 2022-07-19 at 5 50 03 PM](https://user-images.githubusercontent.com/2068657/179854851-7d1427e3-4e65-42ba-90df-5c0f92ea0c54.png)

> There are a lot of tutorials outside of Microsoft documentation, but what's the whole point of having an official documentation at all?

I don't follow... this is the official MSFT SPFx docs and the official tutorials. What you reference is a simple typo that can be fixed...

Thanks for the link... I'll get this fixed.

**Comment by andrewconnell** [comment]
Actually, appears the doc was already update since this issue was initially opened, so there's nothing to fix as it was already fixed in 1ca88faa298ee0d8a29a32d81888314c9b1b1dbb
